docs/en_us/platform_api/source/courses/overview.rst

```diff
@@ -0,0 +1,64 @@
+.. _Courses API Overview:
+
+#############################
+Courses API Overview
+#############################
+
+Use the Courses API to obtain information about edX courses.
```

Comment:
I think we should use lower case for "courses API." It's not an edX trademark, just a component of the software.